Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,19 @@
+/*
 function sum(elements) {
 }
 
 module.exports = sum;
+*/
+
+// ____________________________SOLUTION OR IMPLEMENTATION OF SUM __________________________________
+
+function sum(elements) {
+    // I am using Array.prototype.filter to isolate numbers from the array.
+    // and Array.prototype.reduce to calculate the total sum of the numeric elements.
+
+    return elements
+      .filter((el) => typeof el === "number")
+      .reduce((total, num) => total + num, 0);   // An empty array should return 0.
+  }
+  
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```